java/runtime/src/main/java/org/ray/runtime/task/NativeTaskSubmitter.java

```diff
@@ -37,7 +37,7 @@ public RayActor createActor(FunctionDescriptor functionDescriptor, List<Function
       ActorCreationOptions options) {
     byte[] actorId = nativeCreateActor(nativeCoreWorkerPointer, functionDescriptor, args,
         options);
-    return new NativeRayActor(nativeCoreWorkerPointer, actorId);
+    return NativeRayActor.create(nativeCoreWorkerPointer, actorId);
```

Comment:
We can tell the language from `functionDescriptor`, so it's not necessary to look up the language again in C++ (`nativeGetLanguage`).

Reply:
Good point. I've updated the code and removed `nativeGetLanguage`.